vignettes/customise.Rmd

```diff
@@ -233,6 +233,20 @@ template:
   math-rendering: katex
 ```
 
+On Linux the default symbol font is _Standard Symbols PS_. Unfortunately, this font does not render plotmath expressions correctly under pkgdown's default graphics device `ragg::png`. There are several simple solutions to this:
+
+* Before building your site, use the systemfonts package to change the symbol font to one which does render plotmath expressions correctly under ragg such as _Liberation Sans_, e.g.: 
+  
+  ```r
+  systemfonts::register_variant("symbol", "Liberation Sans")
+  ```
```

Comment:
please add a blank line after code block

Reply:
Added in 88106be.